.gitignore

```diff
@@ -40,7 +40,6 @@ Untitled*.ipynb
 *.bak
 
 # data files 
-*.csv
```

Comment:
I don't have a strong opinion about this, but I also can't see what necessitated it.